docs/sriov.md

```diff
@@ -284,6 +284,18 @@ all this to work. More details on usage can be found in
 and [SR-IOV operator](https://github.com/openshift/sriov-network-operator/blob/master/doc/quickstart.md)
 user documentation.
 
+> **NOTE:**  When using openshift SR-IOV operator,
+> The `NetworkAttachmentDefinition` will be  created automatically, 
+> based on the `SriovNetwork` object. You could also not use the operator. 
+> In that case configuration of the network is done by providing a custom `NetworkAttachmentDefinition`.  
+> There could be some differences between the two methods.
+> One difference that should be noted in particular is that when openshift SR-IOV operator is being used,
+> if VLAN will not be provided in the `SriovNetwork`, the VLAN-ID will be cleared from the SR-IOV-VF.
+> On the other hand When just `NetworkAttachmentDefinition` is used,
+> if VLAN is not specified, the VLAN-ID already configured on the SR-IOV-VF will stay intact, 
+> and in order to clear the VLAN-ID from the VF, 
+> one needs to explicitly configure `vlan=0` on the `NetworkAttachmentDefinition`.
```

Comment:
I find the skeleton here confusing.
It is hard to explain this without entering the needed details.

Therefore, I suggest the note will include the following sections:
- The SR-IOV DP/CNI semantics about the VLAN ID when processing the NAD.
- The SR-IOV Operator `SriovNetwork` object translation to the NAD, which sets the VLAN ID explicitly to 0 if missing (if I understood this correctly).
- The implications when the NAD is used directly and not through the Operator abstraction and how to solve it (set the VLAN ID to 0 explicitly).

I think that if you organize it in this order, the reader will understand what is going on here.